student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
   def play_hooky
     "¯\(ツ)/¯"
   end
+
```

Comment:
- The line `class Student < person` is throwing an error. Is it `person` or `Person`? Please fix that. Make sure classes are imported correctly.